exe/miq

```diff
@@ -124,6 +132,9 @@ module Miq
     desc "build <all|guides|site|reference>", "Build or process an aspect of the site"
     subcommand "build", Build
 
+    desc "generate lwimiq YYYY-MM-DD", "Generate a new blog post"
```

Comment:
:cake: is a lie :)...

```
$ exe/miq generate lwimiq 2017-05-15
ERROR: "miq lwimiq" was called with arguments ["2017-05-15"]
Usage: "miq lwimiq"
```

(BTW, it would also be nice to get the generated filename output on stdout..)